db/db.js

```diff
@@ -401,6 +447,34 @@ async function getLocationIDFromParticleCoreID(coreID) {
   }
 }
 
+async function getRadarTypeFromLocationID(locationid) {
```

Comment:
I don't see this function being used anywhere.

Also, I think that you don't need it. You could just use the `db.getLocationData` method and looks at its `radar_type`, right?